statsforecast/core.py

```diff
@@ -1546,6 +1551,113 @@ def plot(
             palette="tab20b",
         )
 
+    def save(
+        self,
+        path: Union[os.PathLike, str],
+        file_name: Union[os.PathLike, str, None] = None,
+        max_size: str = "50MB",
+        trim: bool = True,
+        trim_attr: list = ["fcst_fitted_values_", "cv_fitted_values_"],
```

Comment:
This is something that we should decide ourselves, it requires knowledge of the internal structures, so just having trim would be enough.
```suggestion
```

Reply:
Will add as variable inside the function.